.travis.yml

```diff
@@ -10,3 +10,7 @@ notifications:
       - D&ENewsFrameworksTeam@bbc.co.uk
     on_failure: change
     on_success: never
+before_install:
```

Comment:
It's a shame we have to do all of this.

I wonder if we remove bundler 1.5 requirement from all of the Alephant gems if they would just work with 2.0. I see it's in the Travis docs:
> https://docs.travis-ci.com/user/languages/ruby/#bundler-20

Reply:
@samfrench Is it okay if I raise an issue to explore this further starting with this gem?